[2020-06-29 16:36:08,842] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: covid_data_dag.upload_data_mysql_task 2020-06-29T11:02:26.241338+00:00 [queued]>
[2020-06-29 16:36:08,850] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: covid_data_dag.upload_data_mysql_task 2020-06-29T11:02:26.241338+00:00 [queued]>
[2020-06-29 16:36:08,851] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-29 16:36:08,851] {taskinstance.py:880} INFO - Starting attempt 1 of 2
[2020-06-29 16:36:08,851] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-29 16:36:08,859] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): upload_data_mysql_task> on 2020-06-29T11:02:26.241338+00:00
[2020-06-29 16:36:08,864] {standard_task_runner.py:53} INFO - Started process 50810 to run task
[2020-06-29 16:36:08,921] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: covid_data_dag.upload_data_mysql_task 2020-06-29T11:02:26.241338+00:00 [running]> localhost.localdomain
[2020-06-29 16:36:08,940] {logging_mixin.py:112} INFO - [('information_schema',), ('metastore_db',), ('mysql',), ('performance_schema',), ('sys',), ('test',), ('twitter',)]
[2020-06-29 16:36:08,940] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-06-29 16:36:08,944] {taskinstance.py:1052} INFO - Marking task as SUCCESS.dag_id=covid_data_dag, task_id=upload_data_mysql_task, execution_date=20200629T110226, start_date=20200629T110608, end_date=20200629T110608
[2020-06-29 16:36:18,836] {logging_mixin.py:112} INFO - [2020-06-29 16:36:18,835] {local_task_job.py:103} INFO - Task exited with return code 0
